lib/smart_answer_flows/can-i-get-a-british-passport.rb

```diff
@@ -1,75 +0,0 @@
-status :draft
-satisfies_need "100983"
```

Comment:
@tadast: Liz says, "I think we're fine - at least 3 of those would have content besides the smart answer, so if there are any needs in maslow, they'll be covered."